tests/core/test_model.py

```diff
@@ -1616,6 +1618,76 @@ def model2_entrypoint(evaluator: MacroEvaluator) -> str:
     )
 
 
+def test_python_model_decorator_kind() -> None:
+    logger = logging.getLogger("sqlmesh.core.model.decorator")
+
+    # no kind specified -> default View kind
+    @model("default_kind", columns={'"COL"': "int"})
+    def a_model(context):
+        pass
```

Comment:
FYI I believe this behavior is about to change so that python models have a `FULL` kind by default, cc @crericha.